src/main/java/blossom/Blossom.java

```diff
@@ -0,0 +1,134 @@
+package blossom;
+
+import blossom.task.Deadline;
+import blossom.task.Event;
+import blossom.task.Task;
+import blossom.task.Todo;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Blossom {
+    private static final String HORIZONTAL_LINE = "____________________________________________________________";
+    private static final String LOGO =
+            """
+                     _______     .---.       ,-----.       .-'''-.    .-'''-.     ,-----.    ,---.    ,---.\s
+                    \\  ____  \\   | ,_|     .'  .-,  '.    / _     \\  / _     \\  .'  .-,  '.  |    \\  /    |\s
+                    | |    \\ | ,-./  )    / ,-.|  \\ _ \\  (`' )/`--' (`' )/`--' / ,-.|  \\ _ \\ |  ,  \\/  ,  |\s
+                    | |____/ / \\  '_ '`) ;  \\  '_ /  | :(_ o _).   (_ o _).   ;  \\  '_ /  | :|  |\\_   /|  |\s
+                    |   _ _ '.  > (_)  ) |  _`,/ \\ _/  | (_,_). '.  (_,_). '. |  _`,/ \\ _/  ||  _( )_/ |  |\s
+                    |  ( ' )  \\(  .  .-' : (  '\\_/ \\   ;.---.  \\  :.---.  \\  :: (  '\\_/ \\   ;| (_ o _) |  |\s
+                    | (_{;}_) | `-'`-'|___\\ `"/  \\  ) / \\    `-'  |\\    `-'  | \\ `"/  \\  ) / |  (_,_)  |  |\s
+                    |  (_,_)  /  |        \\'. \\_/``".'   \\       /  \\       /   '. \\_/``".'  |  |      |  |\s
+                    /_______.'   `--------`  '-----'      `-...-'    `-...-'      '-----'    '--'      '--'""";
+    private static final ArrayList<Task> LIST_OF_TASKS = new ArrayList<Task>();
+    private static final int LENGTH_OF_TODO = 5;
+    private static final int LENGTH_OF_DEADLINE = 9;
+    private static final int LENGTH_OF_EVENT = 6;
+
+    public static void printItems() {
+        // Print items in order
+        System.out.println("Try hard to get these tasks done~~ ");
+        int orderInList = 1;
+        System.out.println(HORIZONTAL_LINE);
+        for(Task item : LIST_OF_TASKS) {
+            System.out.println(orderInList+ ". " + item.toString());
+            orderInList++;
+        }
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    public static void markAndUnmarkItem(int itemIndex, String action) {
```

Comment:
Maybe a better name for this method might be markOrUnmarkItem since you are doing either or.